includes/wc-core-functions.php

```diff
@@ -1801,10 +1801,25 @@ function wc_ascii_uasort_comparison( $a, $b ) {
 function wc_asort_by_locale( &$data, $locale = '' ) {
 	// Use Collator if PHP Internationalization Functions (php-intl) is available.
 	if ( class_exists( 'Collator' ) ) {
-		$locale   = $locale ? $locale : get_locale();
-		$collator = new Collator( $locale );
-		$collator->asort( $data, Collator::SORT_STRING );
-		return $data;
+		try {
+			$locale   = $locale ? $locale : get_locale();
+			$collator = new Collator( $locale );
+			$collator->asort( $data, Collator::SORT_STRING );
+			return $data;
+		} catch ( IntlException $e ) {
+			// Just skip if some error got caused.
```

Comment:
I see this comment spans multilines, perhaps we could use the coding standards here https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments